config/locales/views/users/en.yml

```diff
@@ -38,6 +36,8 @@ en:
 
   # Miscellaneous
   users:
+    display_name: 'Display name'
```

Comment:
@robert-tan I didn't notice this earlier, but this key should actually be under `models/users/en.yml` as an attribute.

Then you can also update the form to use `User.human_attribute_name`